WiinUPro/Windows/MainWindow.xaml.cs

```diff
@@ -38,6 +40,17 @@ public MainWindow()
 
             WinBtStream.OverrideSharingMode = true;
             WinBtStream.OverridenFileShare = System.IO.FileShare.ReadWrite;
+            trayIcon = new System.Windows.Forms.NotifyIcon();
+            trayIcon.Icon = new System.Drawing.Icon(@"..\..\app.ico");
+            trayIcon.Visible = false;
+            trayIcon.DoubleClick +=

```

Comment:
Right now double-clicking the tray icon opens the window. One could instead change this to only require a single left-click, if that would be preferable.